org.fooyin.fooyin.yaml

```diff
@@ -0,0 +1,106 @@
+app-id: org.fooyin.fooyin
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+command: fooyin
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: '23.08'
+    add-ld-path: .
+finish-args:
+  - '--device=dri'
+  - '--filesystem=xdg-download'
+  - '--filesystem=xdg-music'
+  - '--filesystem=xdg-run/pipewire-0'
+  - '--filesystem=/run/media'
+  - '--own-name=org.mpris.MediaPlayer2.fooyin'
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=fallback-x11'
+  - '--socket=pulseaudio'
+  - '--socket=wayland'
+  - '--system-talk-name=org.freedesktop.UDisks2'
+  - '--talk-name=org.freedesktop.Notifications'
+  - '--talk-name=org.gtk.Notifications'
+  - '--talk-name=org.kde.StatusNotifierWatcher'
+  - '--talk-name=org.mpris.MediaPlayer2.*'
```

Comment:
Why is this necessary?

Reply:
I thought this was needed with own-name based on other YAML I have seen.

Reply:
For the MPRIS notification from the app you only need `--own-name=org.mpris.MediaPlayer2.$appname` if the app uses the app-id there (as it should) you don't even need anything as Flatpak by default allows access to `--own-name=org.mpris.MediaPlayer2.$FLATPAK_ID`

Unless the app is a client the talk name and full control over all interfaces isn't required.

Reply:
Seems to be removed by https://github.com/flathub/flathub/pull/5261/commits/e767076c77290bbe179cb7ce98e2b7837d64dcb5 resolving.